.gitignore

```diff
@@ -92,6 +92,7 @@ node_modules/
 /public
 /stories
 /pkg/webui/locales/.backend
+/device-repository-index
```

Comment:
Since we already have `/data/lorawan-devices`, would it make sense to use `/data/lorawan-devices-index`?